public/javascripts/SVValidate/src/label/Label.js

```diff
@@ -34,13 +34,13 @@ function Label(params) {
     };
 
     var icons = {
-        CurbRamp : 'assets/javascripts/SVLabel/img/admin_label_tool/AdminTool_CurbRamp.png',
-        NoCurbRamp : 'assets/javascripts/SVLabel/img/admin_label_tool/AdminTool_NoCurbRamp.png',
-        Obstacle : 'assets/javascripts/SVLabel/img/admin_label_tool/AdminTool_Obstacle.png',
-        SurfaceProblem : 'assets/javascripts/SVLabel/img/admin_label_tool/AdminTool_SurfaceProblem.png',
-        Other : 'assets/javascripts/SVLabel/img/admin_label_tool/AdminTool_Other.png',
-        Occlusion : 'assets/javascripts/SVLabel/img/admin_label_tool/AdminTool_Other.png',
-        NoSidewalk : 'assets/javascripts/SVLabel/img/admin_label_tool/AdminTool_NoSidewalk.png'
+        CurbRamp : '/assets/javascripts/SVLabel/img/admin_label_tool/AdminTool_CurbRamp.png',
```

Comment:
What did this fix again?

Reply:
This fixed issues with our assets not loading properly. I think originally I had created some endpoint like `0.0.0.0:9000/validate/:labelTypeId` that let us create validation missions for specific label type ids. None of our image assets on the validation interface loaded properly when I did this (it would try looking at `0.0.0.0:9000/something/assets/...` which didn't return anything. Adding the `/` makes sure that we're looking at `0.0.0.0:9000/assets/...`, which is where we expect everything to be.